examples/dataframe/create_write_data.py

```diff
@@ -45,9 +45,9 @@
     # Append via single RecordBatch (Arrow)
     batch_single = pa.record_batch(
         [
-            pa.array([6, 7, 8]),
-            pa.array([0.1, 0.2, 0.3]),
-            pa.array([datetime.now().isoformat()] * 3),
+            pa.array([6, 7, 8], type=pa.int32()),
+            pa.array([0.1, 0.2, 0.3], type=pa.float32()),
+            pa.array([datetime.now() for _ in range(3)], pa.timestamp("ns")),
```

Comment:
nit: It's better to use milliseconds, which matches the DFS `TIMESTAMP` data type. DFS will truncate nanoseconds to milliseconds, and nanoseconds can't represent the entire [0001-01-01,9999-12-31] range.

```suggestion
            pa.array([datetime.now() for _ in range(3)], pa.timestamp("ms")),
```